.github/workflows/ci.yml

```diff
@@ -0,0 +1,40 @@
+name: Build and test
+
+on:
+  pull_request:
+  push:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: ["12", "14", "16"]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup NodeJS ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Install dependencies
+        run: npm ci
+      - name: Run unittests
+        run: npm test
+
+  marked-compatibility:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        marked-version: ["1.0.x", "2.0.x", "3.0.x"]
```

Comment:
To clarify this a bit, I prefer to use the lowest minor version because this package requires for instance 1.1.x both this line and the `peerDependency` should be updated to reflect this. The same case could perhaps be said for the patch version but if one adhers to strict semantic versioning a patch should not matter for functionality.